cmp/grid/columns/Column.ts

```diff
@@ -458,7 +459,7 @@ export class Column {
     editorIsPopup: boolean;
     setValueFn: ColumnSetValueFn;
     getValueFn: ColumnGetValueFn;
-    actions?: RecordActionSpec[];
```

Comment:
should be `Array<RecordAction|RecordActionSpec>` (that's what we use for menuActions)